packages/example-app-base/src/utilities/getLinkColors.ts

```diff
@@ -0,0 +1,30 @@
+import { IRawStyle } from 'office-ui-fabric-react';
+
+/**
+ * Gets a style which should be specific enough to override the UHF link styles.
+ * This should be applied to the root of a Link.
+ * @param rest - Color to use in default/rest state
+ * @param hover - Color to use in hover/active state
+ * @param focus - Color to use in focus state (defaults to `rest`)
+ */
+export function getLinkColors(rest: string, hover: string, focus?: string): IRawStyle {
```

Comment:
is this really the right place to fix this? Shouldn't this be done on uhf overrides on the website?

Reply:
The purpose of example-app-base is to create a base set of components for examples; it really has nothing to do with UHF. In addition to the website, M365 and Ribbon examples use it to render example pages. They don't care about UHF overrides, as we may not if we ever move away from the platform.

Reply:
This is effectively the same place the UHF overrides were until Mary removed them in #14270, so putting them back here is no worse than it was before. The only difference is previously they were in Page.module.scss (and therefore globally applied regardless of the theme--the problem Mary was trying to solve) and now they're being applied here via Customizer (therefore overrideable by theme).

I don't want to put them at a higher scope such as fabric-website (or waste time adding a finer-grained customization mechanism) because they should only apply to `Link`s within the `Page` component--otherwise you have to do even more weird re-overrides like the one in SideRail.

Reply:
ah ok.